BinaryEncoding.md

```diff
@@ -56,6 +56,14 @@ A single-byte unsigned integer indicating a [value type](AstSemantics.md#types).
 * `3` indicating type `f32` 
 * `4` indicating type `f64`
 
+### `inline_signature_type`
+A single-byte unsigned integer indicating a signature. These types are encoded as:
```

Comment:
This will be simpler to explain in terms of a single result time, since the signatures won't be relevant until some future post-MVP version.